specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2022-08-01/apimdocumentations.json

```
@@ -0,0 +1,363 @@
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/documentations": {
+      "get": {
+        "tags": [
+          "Documentation"
+        ],
+        "operationId": "Documentation_List_ByService",
+        "description": "Lists all Documentations of the API Management service instance.",
+        "x-ms-examples": {
+          "ApiManagementListApis": {
+            "$ref": "./examples/ApiManagementListDocumentations.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ServiceNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Paged Result response of Documentations.",
+            "schema": {
+              "$ref": "./definitions.json#/definitions/DocumentationCollection"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./apimanagement.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-odata": "./definitions.json#/definitions/DocumentationContract"
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/documentations/{documentationId}": {
+      "head": {
+        "tags": [
+          "Documentation"
+        ],
+        "operationId": "Documentation_GetEntityTag",
+        "description": "Gets the entity state (Etag) version of the Documentation by its identifier.",
+        "x-ms-examples": {
+          "ApiManagementHeadDocumentation": {
+            "$ref": "./examples/ApiManagementHeadDocumentation.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ServiceNameParameter"
+          },
+          {
+            "$ref": "/apimanagement.json#/parameters/DocumentationIdParameter"
```

Hi @malincrist , 
Here maybe the reason of many tools' failures
```suggestion
            "$ref": "./apimanagement.json#/parameters/DocumentationIdParameter"
```